docs/root/configuration/overview/v2_overview.rst

```diff
@@ -329,6 +329,8 @@ you can run:
 
   bazel run //tools:v1_to_bootstrap <path to v1 JSON/YAML configuration file>
 
+.. _config_overview_v2_management_server:
```

Comment:
Note: This anchor is necessary for changes around SDS API links like this https://github.com/envoyproxy/envoy/pull/4415/files#diff-69b12089d28d35dd255d401d59dbe9b9R42.